tox.ini

```
@@ -16,7 +17,7 @@ deps=
     jsonschema
     osqp~=0.6 ; python_version < "3.10"
     pytest
-    pytest-benchmark
+    pytest-benchmark~=4.0.0
```

Same as for optlang.
```suggestion
    pytest-benchmark~=4.0
```

---

thanks!